src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -336,7 +336,7 @@ public static partial class RoslynInsertionTool
 
                     try
                     {
-                        await QueueBuildPolicy(pullRequest, "CloudBuild - Request RPS");
```

Comment:
@AbhitejJohn I also see `Cloud Build - RPS`. But the comment in the PR says we should be running `Request Perf DDRITs`. Not sure if the `Cloud Build - RPS` would give us faster turnaround?

Reply:
You can't actually trigger `CloudBuild-RPS`. It looks more like a status indicator to me.